pkg/policies/branch/branch.go

```diff
@@ -212,6 +212,15 @@ func check(ctx context.Context, rep repositories, c *github.Client, owner,
 				ds[b] = details{}
 				continue
 			}
+			if rsp != nil && rsp.StatusCode == http.StatusForbidden {
+				// Protection not available
+				pass = false
+				text = text + "Branch Protection enforcement is configured in Allstar, however Branch Protection is not available on this repository. Upgrade to GitHub Pro or make this repository public to enable this feature.\n" +
+					"See: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches for more information.\n" +
+					"Alternatively, disable Branch Protection policy enforcement for this repository in Allstar configuration."
```

Comment:
Can remove this alternatively line, it is bad recommendation?

Reply:
If folks don't want to pay for GitHub pro, they can't setup branch protection on private repos. So they probably don't want Allstar configured to bug them to turn it on.

Reply:
ok, then can frame like
"If this is not feasible, then disable ..."

Alternatively makes it think it is ok to do that. so slight language change, then can keep line. thanks for explaining

Reply:
Yes, sounds great. Applied. Please re-approve.